src/test/java/step2/StringAddCalculatorTest.java

```diff
@@ -0,0 +1,46 @@
+package step2;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static step2.StringAddCalculator.splitAndSum;
+
+class StringAddCalculatorTest {
+
+    @Test
+    public void splitAndSum_null_또는_빈문자() {
```

Comment:
```suggestion
    void splitAndSum_null_또는_빈문자() {
```
junit5부터는 테스트 메서드에 public 접근제어자가 제거되어도 됩니다 :)

https://junit.org/junit5/docs/current/user-guide/#writing-tests-classes-and-methods